### PR TITLE
feat(projects): add instructions for creating a Gradientº Project PS-9449 PS-9406

### PR DESCRIPTION
**Pull request by jaredscheib**
i believe this:
Resolves https://paperspace.atlassian.net/browse/PS-9406
Resolves https://paperspace.atlassian.net/browse/PS-9449

let me know if I've misunderstood anything. i wasn't clear on what 9406 may have wanted that was more than what I've done here.

### CODE REVIEW

File: projects/create-a-project.md [hunk 1]
```diff
@@ -1,10 +1,40 @@
-# Create a Project
+# Create a Gradientº Project
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project that represents a container for running Experiments.
```
Comment (danieljosephparker) [inline projects/create-a-project.md]:
```suggestion
As of 7.0, a Gradientº Project represents a container for running Experiments.
```

File: projects/create-a-project.md [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
 
-## Initializing a New Project
+They can be powered by GradientCI (and GitHub), or alternately they can be "Standalone".
```
Comment (danieljosephparker) [inline projects/create-a-project.md]:
```suggestion
Projects can be powered by GradientCI (and GitHub), or alternatively they can be "Standalone".
```

File: projects/create-a-project.md [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+GradientCI Projects allow you to automatically run experiments simply based on pushing code to a GitHub repository.
+
+Standalone Projects require you to provide the code and run new experiments manually, just like running Jobs prior to 7.0.
+
+## Creating a GradientCI Project via the Paperspace Console
+
+To create a Gradient° Project with continuous integration powered by GradientCI and GitHub:</p>
+
+1. Install the [GradientCI](https://github.com/apps/gradientci) GitHub app on your repository. (GitHub Admin privilege is required.)
+1. Navigate to the Projects page.
```
Comment (danieljosephparker) [inline projects/create-a-project.md]:
```suggestion
1. Navigate to the [Projects page](https://www.paperspace.com/console/projects).
```

File: projects/create-a-project.md [hunk 4]
```diff
@@ -1,10 +1,40 @@
...
+1. Click Create Project.
+1. Select GitHub Project.
+1. Grant Paperspace access to your GitHub repos via OAuth.
+1. Confirm a repo with GradientCI installed for your new Gradient° Project.
+
+Voilà! GradientCI will run a new experiment whenever:
+1. a new commit is pushed to the linked repo's default branch;
+1. a PR is opened against that default branch;
+1. a commit is pushed such a PR's branch.
```
Comment (danieljosephparker) [inline projects/create-a-project.md]:
```suggestion
1. a commit is pushed to such a PR's branch.
```

File: projects/create-a-project.md [hunk 5]
```diff
@@ -13,4 +43,3 @@ $ paperspace project init
 ```
 
 This creates a `.ps_project/config.json` file in the current directory to cache information about the project, such as the name, or the last job created.
```
Comment (danieljosephparker) [inline projects/create-a-project.md]:
```suggestion
This creates a `.ps_project/config.yaml` file in the current directory to cache information about the project, such as the name, or the last job created.
```

Comment (danieljosephparker) [reply projects/create-a-project.md]: I believe our config files are only `.yaml` now and not `.json` – @ultrabluewolf can you confirm?

Comment (ultrabluewolf) [reply projects/create-a-project.md]: That's correct, it's yaml.

File: projects/create-a-project.md (hunk 1 above)
Comment (jaredscheib) [reply projects/create-a-project.md]: good catch. i'm actually thinking now that `container` might be a fraught term as well (in the context of docker, etc.). it felt better than `collection`, but maybe that or `group`  is more appropriate. thoughts?

File: projects/create-a-project.md (hunk 2 above)
Comment (jaredscheib) [reply projects/create-a-project.md]: hah, `alternately` and `alternatively` are apparently the same in North America, according to google ;)

File: projects/create-a-project.md [hunk 6]
```diff
@@ -1,16 +1,45 @@
-# Create a Project
+# Create a Gradientº Project
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project represents a container for running Experiments.
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
As of 7.0, a Gradientº Project is a creative workspace for a team or user to run Experiments.
```

File: projects/create-a-project.md [hunk 7]
```diff
@@ -1,16 +1,45 @@
-# Create a Project
+# Create a Gradientº Project
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project is a creative workspace for a team or user to run Experiments.
 
-## Initializing a New Project
+Projects can be powered by GradientCI (and GitHub), or alternatively they can be "Standalone".
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
Projects can be powered by GradientCI (via GitHub), or alternatively they can be "Standalone".
```

File: projects/create-a-project.md [hunk 8]
```diff
@@ -1,16 +1,45 @@
...
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project is a creative workspace for a team or user to run Experiments.
 
-## Initializing a New Project
+Projects can be powered by GradientCI (via GitHub), or alternatively they can be "Standalone".
+
+GradientCI Projects allow you to automatically run experiments simply based on pushing code to a GitHub repository.
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
GradientCI Projects allow you to run Experiments automatically simply by pushing code to a GitHub repository.
```

File: projects/create-a-project.md [hunk 9]
```diff
@@ -1,16 +1,45 @@
...
+
+Standalone Projects require you to provide the code and run new experiments manually, just like running Jobs prior to 7.0.
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
Standalone Projects require you to provide the code and run new Experiments manually, just like running Jobs prior to 7.0.
```

File: projects/create-a-project.md [hunk 10]
```diff
@@ -1,16 +1,45 @@
...
+1. Confirm a repo with GradientCI installed for your new Gradient° Project.
+
+Voilà! GradientCI will run a new experiment whenever:
+1. a new commit is pushed to the linked repo's default branch;
+1. a PR is opened against that default branch;
+1. a commit is pushed to such a PR's branch.
+
+## Creating a Standalone Paperspace Project via the Paperspace Console
+1. Begin the Create Project flow and supply a project name.
+1. Run experiments manually for the Project via the Experiment Builder or [the CLI](https://github.com/Paperspace/paperspace-python).
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
1. Run Experiments manually for the Project via the Experiment Builder or [the CLI](https://github.com/Paperspace/paperspace-python).
```

File: projects/create-a-project.md [hunk 11]
```diff
@@ -1,16 +1,45 @@
...
+1. Grant Paperspace access to your GitHub repos via OAuth.
+1. Confirm a repo with GradientCI installed for your new Gradient° Project.
+
+Voilà! GradientCI will run a new experiment whenever:
+1. a new commit is pushed to the linked repo's default branch;
+1. a PR is opened against that default branch;
+1. a commit is pushed to such a PR's branch.
+
+## Creating a Standalone Paperspace Project via the Paperspace Console
+1. Begin the Create Project flow and supply a project name.
```
Comment (jaredscheib) [inline projects/create-a-project.md]:
```suggestion
1. Begin the Create Project flow and supply a Project name.
```

**Pull request merged**